worlds/alttp/__init__.py

```diff
@@ -24,6 +24,82 @@
 lttp_logger = logging.getLogger("A Link to the Past")
 
 
+class ALTTPWeb(WebWorld):
+    setup_en = Tutorial(
+        "Multiworld Setup Tutorial",
+        "A guide to setting up the Archipelago ALttP Software on your computer. This guide covers single-player, multiworld, and related software.",
+        "English",
+        "multiworld_en.md",
+        "multiworld/en",
+        ["Farrak Kilhn"]
+    )
+
+    setup_de = Tutorial(
+        setup_en.tutorial_name,
+        setup_en.description,
+        "German",
```

Comment:
We should keep the languages written in their own language.
![image](https://user-images.githubusercontent.com/6321333/167223606-f97d8204-1491-4e92-b144-e15c02326d9f.png)

Reply:
So I actually did this at first but in the conversion to json objects the non-unicode characters got mangled and it broke everything. is there an easy way to handle those?